plugins/base/src/test/kotlin/resourceLinks/ResourceLinksTest.kt

```diff
@@ -205,4 +207,93 @@ class ResourceLinksTest : BaseAbstractTest() {
             }
         }
     }
+
+    @Test
+    fun `should not add unknown resources to the end of the head section`() {
```

Comment:
Not completely satisfied with the test as it will break when other resources are added, but couldn't think of anything better to check that nothing else was added to the end :(